### ISSUE TITLE
[Enhancement] Removing "needs-triage" label from closed items via weekly actions

### ISSUE THREAD

**Issue by irvinm** [piroor/treestyletab#3170]
## Abstract
In an effort to keep "needs-triage" labels more relevant, it would be good to add a new section to the "close-expired-issues.yml" to simply remove the "needs-triage" label from closed items.

There are some situations that people resolve their own issues or other members of the community provide help and the items are closed.  By removing the labels, it would allow you to stay more focused on newer items that likely still need your review.

## Expected result
The Github action job would weekly remove any "needs-triage" labels from closed items.

## Actual (present) result
Items that are closed before being triaged still show they should be triaged and clutter the search for that label.

![image](https://user-images.githubusercontent.com/979729/185141900-081c2178-c20a-4947-8f80-30d9e3514844.png)

**Comment by irvinm** [comment]
Would something like this work as a trigger for closed item?  (You would still have to manually remove the existing ones today)

https://github.com/andymckay/labeler
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

```
name: issue-automation

on:
  issues:
    types: [closed]

jobs:
  automate-issues-labels:
    runs-on: ubuntu-latest
    steps:
      - uses: andymckay/labeler@master
        with:
          remove-labels: "needs-triage"
```

**Comment by piroor** [comment]
Thanks! I've added the suggested workflow.

**Comment by piroor** [comment]
I close this to test the workflow.

**Comment by irvinm** [comment]
@piroor looks like it worked!  (Can you remove the "needs-triage" label from the 2 existing closed items?  #3160 and #3155)

**Comment by piroor** [comment]
Thanks, I've done it.
